include/glow/Runtime/Provisioner/Provisioner.h

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_RUNTIME_PROVISIONER_H
+#define GLOW_RUNTIME_PROVISIONER_H
+
+#include <unordered_map>
+
+namespace glow {
+enum ResultCode {
+  READY,
+  EXECUTED,
+  FAILED,
+  CANCELLED
+}; // This will be defined in one common runtime place.
+class executionDAG;
```

Comment:
Could you move the forward declarations right after the line `namespace glow {`? It looks better that way and we usually put forward declarations at the very top.